yarn.lock

```diff
@@ -14148,7 +14128,7 @@ node-int64@^0.4.0:
   version "0.4.0"
   resolved "https://registry.yarnpkg.com/node-int64/-/node-int64-0.4.0.tgz#87a9065cdb355d3182d8f94ce11188b825c68a3b"
 
-node-libs-browser@2.2.1, node-libs-browser@^2.2.1:
+node-libs-browser@^2.2.1:
```

Comment:
Looks like something else depends on `node-libs-browser`?

Reply:
We still install webpack 4 in the monorepo for it's types which has this as a dependency. It shouldn't be being installed with `next` though.